docker-compose.yml

```diff
@@ -3,8 +3,10 @@ version: "3"
 
 services:
     elasticsearch:
-        image: elasticsearch:2.4-alpine
-        platform: linux/amd64
+        image: elasticsearch:7.17.13
```

Comment:
Similarly to my comment on `riotfile.py`, I think this pin needs to remain as old as possible to align with the compatibility policy. To test against ES8, you might consider adding an `elasticsearch8` service to the docker-compose file.

Reply:
Elasticsearch 2 was EOL in 2018, and `7.17` is currently the oldest non-EOL version. I wouldn't have touched this except clients beyond `7.13` will reject connections to older servers as not being recognized as "official".

As you said, we could add new/multiple services to docker-compose, if you feel it's necessary, but my thought was that moving to the oldest compatible version would be sufficient for coverage and preferable for CI workload

Reply:
@emmettbutler I pushed a new commit which runs tests against both es2 and es7 servers. I still feel like this is not needed, it doesn't really increase coverage of anything we're testing with the integration, and all we're really testing is whether the tests themselves are sending what the server expects (and for a 5-years-EOL'd server). In fact I don't even think we need the `opensearch` image some tests run against, those can just be run against es. But not my repo, so your call

Reply:
@emmettbutler Ok no worries, I'll revert that part then. Thanks!